templates/deployment-zabbix-webservice.yaml

```diff
@@ -0,0 +1,79 @@
+{{- if .Values.zabbixwebservice.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "zabbix.fullname" . }}-zabbix-webservice
+  labels:
+    app: {{ template "zabbix.fullname" . }}-zabbix-webservice
+    app.kubernetes.io/name: zabbix-webservice
+    helm.sh/chart: {{ include "zabbix.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-webservice
+    app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-webservice
+spec:
+  replicas: {{ .Values.zabbixwebservice.replicaCount }}
+  selector:
+    matchLabels:
+      app:  {{ template "zabbix.fullname" . }}-zabbix-webservice
+  template:
+    metadata:
+      labels:
+        app: {{ template "zabbix.fullname" . }}-zabbix-webservice
+        app.kubernetes.io/name: zabbix-webservice
+        helm.sh/chart: {{ include "zabbix.chart" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-webservice
+        app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-webservice
+    spec:
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- if gt (len .Values.affinity) 0 }}
+      affinity:
+        {{- toYaml .Values.affinity | nindent 8 }}
+      {{- else if .Values.zabbixwebservice.pod_anti_affinity }}
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 1
+            podAffinityTerm:
+              topologyKey: kubernetes.io/hostname
+              labelSelector:
+                matchLabels:
+                  app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-webservice
+      {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: zabbix-webservice
+        resources:
+          {{- toYaml .Values.zabbixwebservice.resources | nindent 10 }}
+        securityContext:
+          {{- toYaml .Values.securityContext | nindent 12 }}
+        {{- if .Values.zabbix_image_tag }}
+        image: "{{ .Values.zabbixwebservice.image.repository }}:{{ .Values.zabbix_image_tag }}"
+        {{- else }}
+        image: "{{ .Values.zabbixwebservice.image.repository }}:{{ .Values.zabbixwebservice.image.tag }}"
```

Comment:
Hi @sa-ChristianAnton! 

With tag centralized, this line and the if can be removed, correct?

Reply:
commit 3f529c7